File: src/main/java/shinhan/onesol/dto/PaymentMemberDto.java
package shinhan.onesol.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;

@Getter
@AllArgsConstructor
@Builder
public class PaymentMemberDto {
    private Long id;
    private String name;
    private int amount;
    private String cvc;
    private String cardNumber;
    private String cardExpirationYear;
    private String cardExpirationMonth;
    private String customerIdentityNumber;
}

File: frontend/request/PaymentMemberDto.js
class PaymentMemberDto {
  constructor(
    id,
    name,
    amount,
    bankName,
    cardNumber,
    cardExpirationYear,
    cardExpirationMonth,
    customerIdentityNumber
  ) {
    this.id = id;
    this.name = name;
    this.amount = amount;
    this.bankName = bankName;
    this.cardNumber = cardNumber;
    this.cardExpirationYear = cardExpirationYear;
    this.cardExpirationMonth = cardExpirationMonth;
    this.customerIdentityNumber = customerIdentityNumber;
  }
}
export default PayMember; 
